### PR TITLE
Add fallback 'topStoriesTitle' if the top section h2 is empty for the 1st section - Feature Index Page

### LINKED ISSUE #8111: Use the fallback 'topStoriesTitle' translation if the h2 is missing for the 1st section - Features index Page
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by chewu**
Resolves #8111

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by chewu** [comment]
Comparison with localhost and test.
Screenshot 2020-10-15 at 09 38 34

**Review by andrewscfc — APPROVED** [review]
LGTM, not that important for this PR but would be good to populate the PR description with the approach you've taken.

**Review by rhenshaw56 — APPROVED** [review]

File: src/app/containers/IndexPageSection/index.jsx [hunk 1]
```diff
@@ -166,13 +166,16 @@ const sectionBody = ({
...
 
-  const strapline = pathOr('', ['strapline', 'name'], group);
   const isLink = pathOr(null, ['strapline', 'type'], group) === 'LINK';
   const href = pathOr(null, ['strapline', 'links', 'mobile'], group);
   const type = pathOr(null, ['type'], group);
   const seeAll = pathOr(null, ['seeAll'], translations);
   const isFirstSection = sectionNumber === 0;
+  const strapline = isFirstSection
+    ? pathOr(topStoriesTitle, ['strapline', 'name'], group)
+    : pathOr('', ['strapline', 'name'], group); // We are not supposed to default this value if the group is not the 1st section
```
Comment (HarveyPeachey) [inline src/app/containers/IndexPageSection/index.jsx]: This has now overwritten the snapshot that checks for an empty string. Maybe we should re-add a snapshot test for this logic. Or a unit test to test it.

Comment (chewu) [reply src/app/containers/IndexPageSection/index.jsx]: Added 2 snapshot tests for this.

**Review by chris-hinds — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Checked using the URL: https://www.test.bbc.com/pidgin/tori-23133559 and h2 is read out as “Top tori” on NVDA..

LGTM..

**Pull request merged**